ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -38,18 +38,29 @@ object LedgerApiTestTool {
   private def exitCode(summaries: Vector[LedgerTestSummary], expectFailure: Boolean): Int =
     if (summaries.exists(_.result.isLeft) == expectFailure) 0 else 1
 
-  private def printAvailableTests(config: Config): Unit = {
+  private def print[A](defaults: Seq[A], optionals: Seq[A])(t: A => String): Unit = {
     println("Tests marked with * are run by default.")
     println(
       "You can include extra tests with `--include=TEST-NAME`, or run all tests with `--all-tests`.\n")
-    Tests.default.keySet.toSeq.sorted.map(_ + " *").foreach(println(_))
-    Tests.optional(config).keySet.toSeq.sorted.foreach(println(_))
+    defaults.map(t(_) + " * ").sorted.foreach(println(_))
+    optionals.map(t).sorted.foreach(println(_))
 
     println("\nAlternatively, you can run performance tests.")
     println(
       "Performance tests are not run by default, but can be run with `--perf-tests=TEST-NAME`.\n")
     Tests.PerformanceTestsKeys.sorted.foreach(println(_))
   }
+  private def printAvailableTestSuites(config: Config): Unit = {
+    println("Listing test suites. Run with --list-all to see individual tests.")
+    print(Tests.default.values.toSeq, Tests.optional(config).values.toSeq)(_.name)
```

Comment:
Related to the shadowing I mentioned before, reading this in isolation would look a bit weird, given that `print` is defined in the `Prelude`.